File: src/main/scala/org/xbery/artbeams/common/antispam/domain/AntispamQuizFilter.scala
package org.xbery.artbeams.common.antispam.domain

/**
  * @author Radek Beran
  */
case class AntispamQuizFilter(question: Option[String])

object AntispamQuizFilter {
  lazy val Empty = AntispamQuizFilter(None)
}

File: src/main/kotlin/org/xbery/artbeams/common/antispam/domain/AntispamQuizFilter.kt
package org.xbery.artbeams.common.antispam.domain

/**
 * @author Radek Beran
 */
data class AntispamQuizFilter(val question: String?) {
    companion object {
        val Empty: AntispamQuizFilter = AntispamQuizFilter(null)
    }
}
